net/xtables-addons/patches/211-fix-build-with-kernel-6.6.patch

```diff
@@ -0,0 +1,53 @@
+From 03ec7e6466fc73f3278f9c4775348e9739c4abda Mon Sep 17 00:00:00 2001
+From: W_Y_CPP <383152993@qq.com>
+Date: Thu, 25 Jan 2024 22:38:58 -0500
+Subject: [PATCH] 1
+
+---
+ extensions/LUA/prot_buf_ip.c | 4 ++++
```

Comment:
While looking at https://codeberg.org/jengelh/xtables-addons/src/branch/master, I see that these LUA extensions have only been downstream specific for more than 7 years. Is this wanted? Do we still need such extensions? We can not keep this maintenance burden for each kernel version.